### ISSUE TITLE
Syntax error, unrecognized expression on checkboxes

### ISSUE THREAD

**Issue by dallaskoncir** [guillaumepotier/Parsley.js#1002]
Having an issue similar to this one: https://github.com/guillaumepotier/Parsley.js/issues/881

```
<input type="checkbox" name="QID30142||CHECKBOX||95494" id="QID30142||CHECKBOX||95494" />
```

This input gets a data-parsley-multiple="QID30142||CHECKBOX||95494" id="QID30142||CHECKBOX||95494" attribute added, resulting in this error:
Error: Syntax error, unrecognized expression: [data-parsley-multiple=QID30142||CHECKBOX||95492]

I was able to get around this by telling parsley to only look at the email and text inputs. I still got the error when trying to exclude the checkbox inputs though.

**Comment by marcandre** [comment]
Which version are you using?

**Comment by dallaskoncir** [comment]
We are using 2.0.5 on this project.

**Comment by marcandre** [comment]
I'm a bit surprised I have to point out the fact that this was resolved in 2.1.0+, as is written in the issue you already found.

**Comment by dallaskoncir** [comment]
Okay, thanks for the reply. Wasn't sure if that update took care of other characters or just spaces.
